server/src/main/java/org/apache/gravitino/server/web/rest/GroupOperations.java

```diff
@@ -124,6 +130,20 @@ public Response removeGroup(
       return Utils.doAs(
           httpRequest,
           () -> {
+            Owner owner =
+                ownerDispatcher
+                    .getOwner(
+                        metalake, MetadataObjects.of(null, metalake, MetadataObject.Type.METALAKE))
+                    .orElseThrow(
+                        () ->
+                            new IllegalStateException("Owner not found for metalake: " + metalake));
```

Comment:
Does this mean that if the metalake owner cannot be found, we are unable to delete the group as well?